### ISSUE TITLE
Upload from Google drive to wikimedia stuck at 75%

### ISSUE THREAD

**Issue by NdibeRaymond** [tonythomas01/gdrive-to-commons#32]
I tried using this tool on my mobile phone as well as on my laptop and in both cases, it doesn't move beyond 75%. Maybe this has something to do with my account but I would love it if someone can confirm.

**To Reproduce**
1. Open up the tool on either your mobile phone or laptop and try uploading from Google drive to wikimedia

**Expected behavior**
The upload should complete successfully but it isn't

**Image**
![gdrive_to_wikimediacommons_test_drive](https://user-images.githubusercontent.com/40905613/76368579-0401de00-6331-11ea-82ae-4c5764216ed0.JPG)

**Desktop (please complete the following information):**
 - OS: Windows 7 (×86)
 - Browser Chrome
 - Version 80

**Smartphone (please complete the following information):**
 - Device: Gionee M7 power
 - OS: Android 7.1.1
 - Browser Chrome 
 - Version 79

**Comment by NdibeRaymond** [comment]
I would love to work on this @tonythomas01

**Comment by tonythomas01** [comment]
Interesting. Yes, we should fix this. Btw, I would love to see the response from the POST to upload network call to see if we have some error at the backend. 

Also, see that any picture you upload goes directly to Wikimedia Commons.

**Comment by NdibeRaymond** [comment]
@tonythomas01 This issue has been reported in Phabricator by another user so no need duplicating it. I would love to solve it though

**Comment by NdibeRaymond** [comment]
the response from the network is server error 500.
**<h1>Server Error (500)</h1>**

**Comment by sonegillis** [comment]
I had a server error too. I found out that the reason was because I didn't set the correct ```WIKI_URL``` in local_settings.py file. I set it to "test.wikipedia.com" and it gave me another error from google auth which was because I didn't enable google drive api while creating my project on my google account. After enabling, I can now successfully upload
![upload-success](https://user-images.githubusercontent.com/22874347/76854617-b95ff480-684f-11ea-9555-feaf3ef25f3e.png)
